Units/parser-javascript.r/github-issue-1933.d/input.js

```diff
@@ -0,0 +1,11 @@
+testobj = {
+  func1: () => {
+    console.log('test1');
+  },
+  func2: () => {
+    console.log('test2');
+  },
+  func3: x => {
+    console.log(x);
+  }
+};
```

Comment:
As you did in [github-issue-1389.d/input.js](https://github.com/universal-ctags/ctags/pull/3357/commits/f624b4ecd2c48873637dda9ee93e15ffcef7fc5c#diff-bff42b656ff5c564899cc685664805ba3b9d1f35d57c1c5b431f881b53a63cd4), let's add the reference something like:

```js
// Based on #1933 submitted by @gpavlov2016
```